datumaro/plugins/openvino/README.md

```diff
@@ -0,0 +1,75 @@
+# OpenVINO Inference Interpreter 
+Interpreter samples to parse OpenVINO inference outputs.
+
+## Models supported from interpreter samples
+
+- Intel Pre-trained Models
+  - Object Detection
+    - face-detection-0200 (https://docs.openvinotoolkit.org/latest/omz_models_intel_face_detection_0200_description_face_detection_0200.html)
+    - face-detection-0202 (https://docs.openvinotoolkit.org/latest/omz_models_intel_face_detection_0202_description_face_detection_0202.html)
+    - face-detection-0204 (https://docs.openvinotoolkit.org/latest/omz_models_intel_face_detection_0204_description_face_detection_0204.html)
+    - person-detection-0200 (https://docs.openvinotoolkit.org/latest/omz_models_intel_person_detection_0200_description_person_detection_0200.html)
+    - person-detection-0201 (https://docs.openvinotoolkit.org/latest/omz_models_intel_person_detection_0201_description_person_detection_0201.html)
+    - person-detection-0202 (https://docs.openvinotoolkit.org/latest/omz_models_intel_person_detection_0202_description_person_detection_0202.html) 
+    - person-vehicle-bike-detection-2000 (https://docs.openvinotoolkit.org/latest/omz_models_intel_person_vehicle_bike_detection_2000_description_person_vehicle_bike_detection_2000.html)
+    - person-vehicle-bike-detection-2001 (https://docs.openvinotoolkit.org/latest/omz_models_intel_person_vehicle_bike_detection_2001_description_person_vehicle_bike_detection_2001.html)
+    - person-vehicle-bike-detection-2002 (https://docs.openvinotoolkit.org/latest/omz_models_intel_person_vehicle_bike_detection_2002_description_person_vehicle_bike_detection_2002.html)
+    - vehicle-detection-0200 (https://docs.openvinotoolkit.org/latest/omz_models_intel_vehicle_detection_0200_description_vehicle_detection_0200.html)
+    - vehicle-detection-0201 (https://docs.openvinotoolkit.org/latest/omz_models_intel_vehicle_detection_0201_description_vehicle_detection_0201.html)
+    - vehicle-detection-0202 (https://docs.openvinotoolkit.org/latest/omz_models_intel_vehicle_detection_0202_description_vehicle_detection_0202.html)
+
+- Public Pre-Trained Models(OMZ)
+  - Classification
```

Comment:
Probably, it would be great, if you gave a notion about classes of supported models, instead of specific models. This would also cover custom, manually-converted models.

Reply:
can you give an example? The model link contains model input/output info. Do you want to change,
before
  - Classification
    - [mobilenet-v2-pytorch](https://docs.openvinotoolkit.org/latest/omz_models_public_mobilenet_v2_pytorch_mobilenet_v2_pytorch.html)


to
  - Classification
    - [mobilenet-v2-pytorch](https://docs.openvinotoolkit.org/latest/omz_models_public_mobilenet_v2_pytorch_mobilenet_v2_pytorch.html): imagenet 1000 classes

Reply:
No, I'm speaking about a line "or any other model with the last layer of type DetectionOutput" or any other similar kind of advice for searching other classes of models, which can be supported, but not tested (including custom user models).